tictactoe.py

```diff
@@ -0,0 +1,73 @@
+class Tic_Tac_Toe:
```

Comment:
you want to run this entire file through a py linter, clean it up before review
https://pypi.org/project/pep8/